enforcer.go

```diff
@@ -132,7 +135,20 @@ func NewEnforcer(params ...interface{}) (*Enforcer, error) {
 
 // InitWithFile initializes an enforcer with a model file and a policy file.
 func (e *Enforcer) InitWithFile(modelPath string, policyPath string) error {
-	a := fileadapter.NewAdapter(policyPath)
+	var a persist.Adapter
+
+	// When the policy path is empty, the user only passes the model path.
+	// If the policy path is not empty, we have to make sure the file exists.
+	if policyPath != "" {
+		_, err := os.Stat(policyPath)
```

Comment:
Use this util function:

```go
func FileExist(path string) bool {
	if _, err := os.Stat(path); os.IsNotExist(err) {
		return false
	}
	return true
}
```

Maybe put it into a util package or util.go

Reply:
good point.